core_ui/lib/src/widgets/main_screen/mobile/report_statistics.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:api_client/api_client.dart';
+import 'package:flutter/material.dart';
+
+class ReportStatistics extends StatelessWidget {
+  const ReportStatistics({
+    super.key,
+    required this.width,
+    required this.reportStatistics,
+    required this.isMobile,
+  });
+
+  final double width;
+  final ReportStatisticsDto reportStatistics;
+  final bool isMobile;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: width,
+      child: isMobile
+          ? Column(children: getStatWidgets(reportStatistics))
+          : Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: getStatWidgets(reportStatistics)),
+    );
+  }
+
+  List<Widget> getStatWidgets(ReportStatisticsDto stats) {
+    List<Widget> statWidgets = [];
```

Comment:
For enhanced code readability and simplicity, consider creating a custom widget for `ReportTypeStatistics` and passing it directly as a child to the `Wrap` widget. This approach not only streamlines the code but also makes it easier to manage and maintain.

Additionally, as a bonus improvement, transforming the 'status' property into an enum could further simplify the report-building process. Although this enhancement is beyond the scope of the current pull request, it presents an opportunity to optimize the codebase in the future. By utilizing an enum for 'status' and implementing switch cases within the custom widget, we can enhance maintainability and extend functionality seamlessly.